users/strelaysrv.rst

```diff
@@ -0,0 +1,142 @@
+.. _strelaysrv:
+
+Syncthing Relay Server
+======================
+
+Synopsis
+--------
+
+::
+
+    strelaysrv [-debug] [-ext-address=<address>] [-global-rate=<bytes/s>] [-keys=<dir>] [-listen=<listen addr>]
+               [-message-timeout=<duration>] [-network-timeout=<duration>] [-per-session-rate=<bytes/s>]
+               [-ping-interval=<duration>] [-pools=<pool addresses>] [-provided-by=<string>] [-status-srv=<listen addr>]
+
+Description
+-----------
+
+Syncthing relies on a network of community-contributed relay servers. Anyone
+can run a relay server, and it will automatically join the relay pool and be
+available to Syncthing users. The current list of relays can be found at
+https://relays.syncthing.net.
+
+Options
+-------
+
+.. cmdoption:: -debug
+
+    Enable debug output.
+
+.. cmdoption:: -ext-address=<address>
+
+    An optional address to advertising as being available on. Allows listening
```

Comment:
s/to advertising/to advertise
The typo/grammar was there in the previous text as well, and is in the source code.
Source code text fix proposed in https://github.com/syncthing/syncthing/pull/3377